jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -1398,108 +1398,108 @@
         trigger-job: 'ci-kubernetes-build-1.6'
 
     # gke-version-pinned
-    # trigger-job: 'ci-kubernetes-build-1.4'
+    # trigger-job: 'ci-kubernetes-build-1.6'
     # TODO(spxtr) float with current release.
     - kubernetes-e2e-gke-pre-release:
         job-name: ci-kubernetes-e2e-gke-pre-release
         jenkins-timeout: 600
         timeout: 500
         frequency: 'H/5 * * * *' # At least every 30m
-        trigger-job: 'ci-kubernetes-build-1.4'
+        trigger-job: 'ci-kubernetes-build-1.6'
     - kubernetes-e2e-gke-test:
         job-name: ci-kubernetes-e2e-gke-test
         jenkins-timeout: 600
         timeout: 500
         frequency: 'H/5 * * * *' # At least every 30m
-        trigger-job: 'ci-kubernetes-build-1.4'
+        trigger-job: 'ci-kubernetes-build-1.6'
     - kubernetes-e2e-gke-subnet:
         job-name: ci-kubernetes-e2e-gke-subnet
         jenkins-timeout: 600
         timeout: 500
         frequency: 'H/5 * * * *' # At least every 30m
-        trigger-job: 'ci-kubernetes-build-1.4'
+        trigger-job: 'ci-kubernetes-build-1.6'
     - kubernetes-e2e-gke-staging:
         job-name: ci-kubernetes-e2e-gke-staging
         jenkins-timeout: 600
         timeout: 500
         frequency: 'H/5 * * * *' # At least every 30m
-        trigger-job: 'ci-kubernetes-build-1.4'
+        trigger-job: 'ci-kubernetes-build-1.6'
     - kubernetes-e2e-gke-staging-parallel:
         job-name: ci-kubernetes-e2e-gke-staging-parallel
         jenkins-timeout: 200
         timeout: 100
         frequency: 'H/5 * * * *' # At least every 30m
-        trigger-job: 'ci-kubernetes-build-1.4'
+        trigger-job: 'ci-kubernetes-build-1.6'
     - kubernetes-e2e-gke-prod:
         job-name: ci-kubernetes-e2e-gke-prod
         jenkins-timeout: 600
         timeout: 500
         frequency: 'H/5 * * * *' # At least every 30m
-        trigger-job: 'ci-kubernetes-build-1.4'
+        trigger-job: 'ci-kubernetes-build-1.6'
```

Comment:
These jobs run frequently enough that the trigger job really has no effect, anyway.